pkg/svcgen/testing/run.go

```diff
@@ -0,0 +1,100 @@
+package testing
+
+import (
+	"go/build"
+	"testing"
+
+	"github.com/bradleyjkemp/cupaloy"
+	"github.com/izumin5210/grapi/pkg/cli"
+	"github.com/izumin5210/grapi/pkg/gencmd"
+	"github.com/izumin5210/grapi/pkg/grapicmd/util/fs"
+	"github.com/spf13/afero"
+)
+
+type Ctx struct {
+	GOPATH    string
+	RootDir   cli.RootDir
+	CreateCmd func(*testing.T, afero.Fs, Case) gencmd.Cmd
+	Cases     []Case
+}
+
+type Case struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Case should have comment or be unexported